public/app/plugins/panel/flamegraph/components/FlameGraph/rendering.ts

```diff
@@ -101,19 +101,17 @@ export function renderRect(
   const l = 65 + 7 * intensity;
 
   const name = rect.label;
-  const idxs = ufuzzy.filter([name], query);
```

Comment:
@joey-grafana it's much cheaper to do this filtering once on big haystack than 100k times on a haystacks of 1 item.